smoke-test/src/test/java/org/opennms/smoketest/NodeMapIT.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2019 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2019 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.smoketest;
+
+import static com.jayway.awaitility.Awaitility.with;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+import org.junit.Test;
+
+public class NodeMapIT extends OpenNMSSeleniumTestCase {
```

Comment:
Is there an existing class we can add this test under instead? (If there isn't it's OK.)

Right now adding a new test class to the smoke tests adds about 4 minutes to every run.

Reply:
There is no node-map related smoke-test yet, but of course this can be added to any other unrelated smoke test since the test method itself isn‘t very complex. What do you think?

Reply:
Yeah, I can't see any other good place to put it for now - let's keep it as-is then.

We'll rework these as a separate effort.